shared/base/view.js

```diff
@@ -195,7 +195,7 @@ module.exports = BaseView = Backbone.View.extend({
    * Get the HTML for the view, including the wrapper element.
    */
   getHtml: function() {
-    var html = this.getInnerHtml(),
+    var html = this.options.lazy ? '' : this.getInnerHtml(),
```

Comment:
Sort of related, but I'm a bit unclear about the intent of getHtml. Here https://github.com/rendrjs/rendr-handlebars/blob/master/shared/helpers/view.js#L55 in the rendr-handlebars code, the comment makes it seem like view.getHtml should only be used on the server to return the outer wrapper element, but without this change, it always tries to insert the inner html as well.